nautobot/extras/management/commands/runjob.py

```diff
@@ -53,6 +55,13 @@ def handle(self, *args, **options):
             request.id = uuid.uuid4()
             request.user = user
 
+        data = {}
+        try:
+            if options.get("data"):
+                data = json.loads(options["data"])
```

Comment:
```suggestion
        try:
            data = json.loads(options.get("data", "{}"))
```

Reply:
This is not required, if you look into the rest of the file, you will see that despite the same argparse pattern there is no .get on the key, here I printed out the key.


```
nautobot-server runjob plugins/example_plugin.jobs/ExampleLoggingJob --commit -u admin
{
    "verbosity": 1,
    "settings": None,
    "pythonpath": None,
    "traceback": False,
    "no_color": False,
    "force_color": False,
    "skip_checks": False,
    "job": "plugins/example_plugin.jobs/ExampleLoggingJob",
    "commit": True,
    "username": "admin",
    "data": None,
}
```

Reply:
Personally I prefer Gary's suggestion as well - it's more succinct and has the same result.